packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -4,7 +4,8 @@ import useWindowSize from "src/hooks/useWindowSize";
 import Desktop from "./Menu/Desktop";
 import Mobile from "./Menu/Mobile";
 import { IconType } from "react-icons/lib";
-import { MdHorizontalSplit, MdSpaceDashboard } from "react-icons/md";
+import { MdHorizontalSplit, MdLoop, MdSpaceDashboard } from "react-icons/md";
```

Comment:
Does this takin all of "md" or has react-icons started tree shaking?

Reply:
might be an old issue https://github.com/react-icons/react-icons/issues/154, so probably worth adding an item to check how much icons we're loading

Reply:
add a ticket to the debt board for us to look into how much icons we're importing

Reply:
yeah good point, will add!